extensions/wikia/AdEngine/js/SlotTweaker.js

```diff
@@ -155,7 +189,7 @@ define('ext.wikia.adEngine.slotTweaker', [
 		});
 	}
 
-	function noop() {
+	function noop(arg) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "arg". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)